collector/src/main/java/com/navercorp/pinpoint/collector/service/TraceService.java

```diff
@@ -177,12 +181,21 @@ private void insertSpanEventStat(SpanBo span) {
     }
 
     private void insertSpanEventList(List<SpanEventBo> spanEventList, ServiceType applicationServiceType, String applicationId, String agentId, String endPoint) {
+
         for (SpanEventBo spanEvent : spanEventList) {
             final ServiceType spanEventType = registry.findServiceType(spanEvent.getServiceType());
+
+            if (spanEventType.isAlias()) {
```

Comment:
If aliased **ServiceType** needs to be `recordStatistics=true` as well, you might want to place this check after `if (!spanEventType.isRecordStatistics())`.
If it doesn't, please ignore this comment.

Reply:
aliased ServiceType need to be`recordStatistics=false`